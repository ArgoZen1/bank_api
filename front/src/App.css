html {
  font-family: Avenir, Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  /* stylelint-disable-next-line order/properties-order */
  color: #2c3e50;
}
