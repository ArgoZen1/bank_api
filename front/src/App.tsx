import NavBar from './Components/NavBar/NavBar';
import { BrowserRouter, Routes, Route } from 'react-router-dom';
import Home from './Components/Home/Home';
import SignIn from './Components/SignIn/SignIn';
import User from './Components/User/User';


function App() {
  return (
    <BrowserRouter>
      <NavBar />
      <Routes>
        <Route path="/" element={<Home />} />
        <Route path="/login" element={<SignIn />} />
        <Route path="/user" element={<User />} />
      </Routes>
    </BrowserRouter>

  );
}

export default App;