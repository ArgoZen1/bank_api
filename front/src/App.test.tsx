import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import App from './App'

describe('App component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});