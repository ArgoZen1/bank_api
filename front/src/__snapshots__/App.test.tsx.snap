// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component should render correctly 1`] = `
<DocumentFragment>
  <nav
    class="main-nav"
  >
    <a
      class="main-nav-logo"
      href="/"
    >
      <img
        alt="Logo"
      />
    </a>
    <div>
      <a
        class="main-nav-item"
        href="/login"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-circle-user icon_user"
          data-icon="circle-user"
          data-prefix="fas"
          data-testid="icon_user"
          focusable="false"
          role="img"
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M399 384.2C376.9 345.8 335.4 320 288 320H224c-47.4 0-88.9 25.8-111 64.2c35.2 39.2 86.2 63.8 143 63.8s107.8-24.7 143-63.8zM512 256c0 141.4-114.6 256-256 256S0 397.4 0 256S114.6 0 256 0S512 114.6 512 256zM256 272c39.8 0 72-32.2 72-72s-32.2-72-72-72s-72 32.2-72 72s32.2 72 72 72z"
            fill="currentColor"
          />
        </svg>
        Sign In
      </a>
    </div>
  </nav>
  <main>
    <div
      class="hero"
    >
      <section
        class="hero-content"
      >
        <h2
          class="sr-only"
        >
          Promoted Content
        </h2>
        <p
          class="subtitle"
        >
          No fees.
        </p>
        <p
          class="subtitle"
        >
          No minimum deposit.
        </p>
        <p
          class="subtitle"
        >
          High interest rates.
        </p>
        <p
          class="text"
        >
          Open a savings account with Argent Bank today!
        </p>
      </section>
    </div>
    <section
      class="features"
    >
      <h2
        class="sr-only"
      >
        Features
      </h2>
      <div
        class="feature-item"
      >
        <img
          alt="Chat Icon"
          class="feature-icon"
        />
        <h3
          class="feature-item-title"
        >
          You are our #1 priority
        </h3>
        <p>
          Need to talk to a representative? You can get in touch through our 24/7 chat or through a phone call in less than 5 minutes.
        </p>
      </div>
      <div
        class="feature-item"
      >
        <img
          alt="Money Icon"
          class="feature-icon"
        />
        <h3
          class="feature-item-title"
        >
          More savings means higher rates
        </h3>
        <p>
          The more you save with us, the higher your interest rate will be!
        </p>
      </div>
      <div
        class="feature-item"
      >
        <img
          alt="Security Icon"
          class="feature-icon"
        />
        <h3
          class="feature-item-title"
        >
          Security you can trust
        </h3>
        <p>
          We use top of the line encryption to make sure your data and money is always safe.
        </p>
      </div>
    </section>
  </main>
  <footer
    class="footer"
  >
    <p
      class="footer-text"
    >
      Copyright 2020 Argent Bank
    </p>
  </footer>
</DocumentFragment>
`;
